server/src/classes/__tests__/eventtimer.test.js

```diff
@@ -68,8 +68,8 @@ test('object instantiates correctly', async () => {
   expect(t.nextEventId).toBeNull();
   expect(t.selectedPublicEventId).toBeNull();
   expect(t.nextPublicEventId).toBeNull();
-  expect(t.numEvents).toBe(0);
```

Comment:
Should we also remove it from the source code if we're not testing `numEvents` anymore? I don't see other mentions of this in the PR.

Reply:
We no longer expose numEvents as a class property, but it is useful to know this for testing some of the behaviours.
Unless I am misunderstanding the suggestion, I believe this is a good assertion